lib/pub_cache.dart

```diff
@@ -10,19 +10,16 @@ import 'package:path/path.dart' as path;
 import 'package:pub_semver/pub_semver.dart';
 import 'package:yaml/yaml.dart' as yaml;
 
-// TODO: do things like get all the versions for a package?
-
 // TODO: get the latest version for a package?
 
 // TODO: get the latest non-dev version for a package?
 
-// TODO: get all the packages (id's only) in the pubcache
+// TODO: Scan for git packages.
 
 /**
- * TODO:
+ * A programattic API for reflecting on Pub's cache directory.
```

Comment:
I know you're old fashioned by choice but the [new kids](https://www.dartlang.org/articles/doc-comment-guidelines/) all use `///`'s! :)